app/Settings/HeaderSettings.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace BlazeWooless\Settings;
+
+class HeaderSettings extends BaseSettings {
+	private static $instance = null;
+	public $tab_key = 'header';
+	public $page_label = 'Header';
+	public $setting_page_name = 'blaze-settings-header';
+
+	public static function get_instance() {
+		if ( self::$instance === null ) {
+			self::$instance = new self( 'wooless_header_settings_options' );
+		}
+
+		return self::$instance;
+	}
+
+	public function settings_callback( $options ) {
+		return $options;
+	}
+
+	public function settings() {
+		return array();
+	}
+
+	public function footer_callback() {
+		$args = array(
+			'post_type' => 'blaze_settings',
+			'name' => $this->setting_page_name,
+		);
+		$the_query = new \WP_Query( $args );
+
+		if (!$the_query->have_posts()) {
+			$new_post = array(
+				'post_title' => 'Header',
+				'post_type' => 'blaze_settings',
+				'post_name' => $this->setting_page_name,
+				'post_category' => array(0)
+			);
+			$post_id = wp_insert_post($new_post);
+		} else {
+			$post_id = $the_query->posts[0]->ID;
+		}
```

Comment:
remove else and make this as the default value instead of adding it as an else statement